cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/template/etl/realtime/source/JmsSource.java

```diff
@@ -328,13 +330,28 @@ public JmsProvider getJmsProvider() {
     @Nullable
     private String connectionFactoryName;
 
+    @Name(JMS_PLUGIN_NAME)
+    @Description("Name of the JMS plugin to use. This is the value of the 'name' key defined in the json file " +
+        "for the JMS plugin. Defaults to " + Context.INITIAL_CONTEXT_FACTORY)
+    @Nullable
+    public String jmsPluginName;
```

Comment:
What is the difference between setting it in default constructor vs variable definition?